tests/x11/yast2_lan_restart.pm

```diff
@@ -90,4 +147,8 @@ sub post_fail_hook {
     $self->SUPER::post_fail_hook;
 }
 
+sub test_flags {
+    return {always_rollback => 1} if (check_var('BACKEND', 'qemu'));
```

Comment:
We don't need this check, as always_rollback affects qemu only.

Reply:
Removed